main.go

```diff
@@ -100,6 +101,10 @@ func handleError(s *server.Server, message string, err error) {
 
 func runMemphis(s *server.Server) {
 
+	if !s.MemphisInitialized() {
+		s.Fatalf("Jetstream not enabled on global account")
```

Comment:
what this error means

Reply:
that we started NATS w/o enabling Jetstream, 
mostly internal as we should always use NATS+JS, 
a good point, we should make the default behaviour to run with the --js flag